app.py

```
@@ -11,7 +11,7 @@
 import concurrent.futures
 from google.cloud import speech, texttospeech
 import logging
-from flask import Flask, request, jsonify, render_template
+from flask import Flask, request, jsonify, render_template, redirect
```

Ruff (0.8.2)

`flask.redirect` imported but unused

Remove unused import: `flask.redirect`

(F401)